docs/core/diagnostics/built-in-metrics-system-net.md

```diff
@@ -0,0 +1,138 @@
+---
+title: System.Net Metrics
+description: Review the metrics available for System.Net
+ms.topic: reference
+ms.date: 9/21/2023
+---
+
+# System.Net Metrics
+
+This article describes the networking metrics built-in for <xref:System.Net> produced using the
+<xref:System.Diagnostics.Metrics?displayProperty=nameWithType> API. For a listing of metrics based on the alternate [EventCounters](event-counters.md) API,
+see [here](available-counters.md).
+
+- [Meter: `System.Net.NameResolution`](#meter-systemnetnameresolution) - Metrics for DNS lookups
+  * [Instrument: `dns.lookup.duration`](#instrument-dnslookupduration)
+- [Meter: `System.Net.Http`](#meter-systemnethttp) - Metrics for outbound networking requests using HttpClient
+  * [Instrument: `http.client.open_connections`](#instrument-httpclientopen_connections)
+  * [Instrument: `http.client.connection.duration`](#instrument-httpclientconnectionduration)
+  * [Instrument: `http.client.request.duration`](#instrument-httpclientrequestduration)
+  * [Instrument: `http.client.request.time_in_queue`](#instrument-httpclientrequesttime_in_queue)
+  * [Instrument: `http.client.active_requests`](#instrument-httpclientactive_requests)
+
+## Meter: `System.Net.NameResolution`
+
+### Instrument: `dns.lookup.duration`
+
+| Name     | Instrument Type | Unit | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `dns.lookup.duration` | Histogram | `s` | Measures the time taken to perform a DNS lookup. |
+
+| Attribute  | Type | Description  | Examples  | Presence |
+|---|---|---|---|---|
+| `dns.question.name` | string | The name being queried. | `www.example.com`; `dot.net` | Always |
```

Comment:
We should add `error.type` when https://github.com/dotnet/runtime/pull/93254 is backported.

Reply:
Added